recipes/rna-cd/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "rna-cd" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rna_cd-{{ version }}.tar.gz
+  sha256: a88bacede091319da4879f79b9688aea7f5a12f7f5107b3850804fed0d7044cc
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - rna_cd-train = rna_cd.cli:train_cli
+    - rna_cd-classify = rna_cd.cli:classify_cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - click
+    - joblib
+    - matplotlib
+    - pip
+    - pysam
+    - python >=3.5
+    - scikit-learn
+  run:
+    - click
+    - joblib
+    - matplotlib
+    - pysam
+    - python >=3.5
+    - scikit-learn
+
+test:
+  imports:
+    - rna_cd
+  commands:
+    - rna_cd-train --help
+    - rna_cd-classify --help
+
+about:
+  home: https://github.com/LUMC/rna_cd
+  license: GNU Affero General Public v3 or later (AGPLv3+)
+  license_family: AGPL
+  license_file: 
+  summary: RNA contamination detector
+  doc_url: https://rna-cd.readthedocs.io/en/latest/ 
+  dev_url: https://github.com/LUMC/rna_cd
+
+extra:
+  recipe-maintainers:
+    - sndrtj
```

Comment:
Can you remove the empty `license_file:` line? You can also remove the `extra:` section if you like, we don't actually use it for anything (only conda-forge does). Other than that it looks good!

Reply:
Thanks for the review! This is done in commit 80f1edb.